src/components/Title.jsx

```diff
@@ -14,7 +14,7 @@ const Title = ({ text, food, color }) => {
     <div className="flex flex-row justify-center">
       <div className="inset-0 flex justify-center items-top p-3">
         <div
-          className={`text-shadow ${darkColor} font-omc text-8xl mb-2 drop-shadow left-1 bottom-0.5 ${lightColor}`}
+          className={`text-shadow ${darkColor} font-omc text-5xl md:text-8xl mb-2 drop-shadow left-1 bottom-0.5 ${lightColor} text-center`}
         >
           {text}
         </div>
```

Comment:
The food image still needs to be smaller on mobile

I think in the Image src, its passing in the image as a filepath string (check data file for COOKIES).

Import the images in COOKIES to get rid of the width and height, then use tailwind width and height to change the size of the image (making it more mobile responsive).

Reply:
if the above suggestion isn't possible just remove cookie from mobile view. ^ 

you can do so by using hidden